src/riak_cs_gc_key_list.erl

```diff
@@ -180,6 +182,46 @@ gc_index_query(RcPid, StartKey, EndKey, BatchSize, Continuation, UsePaginatedInd
 
     {QueryResult, {StartKey, EndKey}}.
 
+-spec find_oldest_entries(BagId::binary()|master) ->
+                                 {ok, [{string(), [char()]}]} |
```

Comment:
`char()` here seems a little strange for me, because it is to represent unicode code points. For plain integers , `interger()` (or maybe `non_neg_integer()`) is generally used. If emphasis of "small integers" is preferable, `byte()` may be good too.